examples/quota-increase/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  # This module is now only being tested with Terraform 0.13.x. However, to make upgrading easier, we are setting
+  # 0.12.26 as the minimum version, as that version added support for required_providers with source URLs, making it
+  # forwards compatible with 0.13.x code.
+  required_version = ">= 0.12.26"
+}
+
+provider "aws" {
+  region = var.aws_region
+}
+
+module "quota-increase" {
+  source = "../../modules/quota-increase"
+
+  resources_to_increase = {
+    nacl_rules = 30
```

Comment:
What happens when you run `destroy`? Does the quota reset to the old value? 

Or what happens if the quota is already at or above the requested value? Do you get an error?

Reply:
Based on the `aws service-quotas list-requested-service-quota-change-history`:

1. Running `destroy` does nothing! It just cleans the state.
1. If the quota requested is smaller than the current quota, it runs successfully and does not create a new request! The output will have the current value.
  e.g. NAT Gateway current quota is 30. I ask for 25 quota, this is the output:
   ```
   new_quotas = {
    "nat_gateway" = {
      "adjustable" = true
      "arn" = "arn:aws:servicequotas:us-east-1:<account-id>:vpc/L-FE5A380F"
      "default_value" = 5
      "id" = "vpc/L-FE5A380F"
      "quota_code" = "L-FE5A380F"
      "quota_name" = "NAT gateways per Availability Zone"
      "service_code" = "vpc"
      "service_name" = "Amazon Virtual Private Cloud (Amazon VPC)"
      "value" = 30   <------ Returned the current quota, not the requested one.
    }
   }
   ```
   Also, there is no new item on the history.
   
Added a section for this on the docs at ec1e29a

Reply:
Roger, thanks!